client/src/containers/approver/people/People.js

```diff
@@ -153,6 +154,11 @@ class People extends Component {
                   handleUserSelect={this.handleUserSelect}
                   {...this.props}/>
               }
+              { activeIndex === 2 &&
```

Comment:
After clicking on View all roles button, I only see two entries on the list of Roles page. After clicking Load More, it shows me 2 more roles. Should we increase the size of the page to maybe 5 or 10? Some users might have over 30 roles and they might want to quickly view their roles.

Reply:
Updated to 10.